### ISSUE TITLE
EasyEffects Sink is set as default on port change

### ISSUE THREAD

**Issue by SteelT1** [wwmm/easyeffects#1916]
### EasyEffects Version

6.3.0

### What package are you using?

Arch (easyeffects)

### Distribution

Arch Linux

### Describe the bug

When the port in use is changed, i.e speakers to headphones, the default output gets changed from the actual output device to EasyEffects Sink.

### Expected Behavior

Default output device stays the same when a port change occurs.

**Comment by wwmm** [comment]
As easyeffects does not try to set default devices this is probably a pipewire or wire plumber bug. There is nothing we can do on our side.

**Comment by SteelT1** [comment]
Quick testing shows that it happens with wireplumber and not when using the deprecated media-session, and made an issue of the findings on the wirepulmber repository

**Comment by shrx** [comment]
This is my workaround for this issue:

1) write a script that sets the desired audio sink and source

2) trigger the script whenever an "audio jack connection" event is detected by acpid

The script (ugly due to workarounds to make `pactl` commands work when executed as root), located in `/etc/acpi/handlers/switch-to-builtin-audio`:

```bash
#!/usr/bin/bash

USER=<your username>

export XAUTHORITY=/home/$USER/.Xauthority

SINK=$(  su $USER -c "/usr/bin/pactl list short sinks  |grep analog-stereo|grep -v monitor")
SOURCE=$(su $USER -c "/usr/bin/pactl list short sources|grep analog-stereo|grep -v monitor")
SINK=$(  echo $SINK | awk '{print $2}')
SOURCE=$(echo $SINK | awk '{print $2}')

su $USER -c "/usr/bin/pactl set-default-sink $SINK"
su $USER -c "/usr/bin/pactl set-default-source $SOURCE"
```

The acpi event rule (in `/etc/acpi/events/override-easyeffects-sink`):

```bash
event=jack/headphone
action=/etc/acpi/handlers/switch-to-builtin-audio
```

**Comment by pinkflames** [comment]
It seems like this is an Xfce bug that allegedly is fixed in xfce4-pulseaudio-plugin 0.4.6.

**Comment by shrx** [comment]
> It seems like this is an Xfce bug that allegedly is fixed in xfce4-pulseaudio-plugin 0.4.6.

I can confirm, compiling and installing xfce4-pulseaudio-plugin 0.4.6 fixes the issue for me.
